hw3/01-characters.js

```diff
@@ -1,4 +1,62 @@
 // url for the Thrones API
-const url = 'https://thronesapi.com/api/v2/Characters';
+const url = "https://thronesapi.com/api/v2/Characters";
 
-fetch(url);
+const charContainer = document.querySelector("#characters");
+const printCharacters = async function () {
+  try {
+    const response = await fetch(url);
+    const jsonData = await response.json();
+
+    charContainer.classList.add("card-deck");
+    jsonData.forEach((element) => {
+      const { fullName, title, family, imageUrl } = element;
+      //card
+      const charDiv = document.createElement("div");
+      charDiv.classList.add("card");
+
+      //card image
+      const charImage = document.createElement("img");
+      charImage.classList.add("card-img");
+      charImage.src = imageUrl;
+
+      if (family.includes("House")) {
+        charImage.alt = `${fullName} of ${family}`;
+      } else {
+        family.includes("Free") |
+        family.includes("Unk") |
+        family.includes("None")
+          ? (charImage.alt = `${fullName} - ${title}`)
+          : (charImage.alt = `${fullName} of House ${family}`);
```

Comment:
I think this can be 

```suggestion
        charImage.alt = family.includes("Free") ||
        family.includes("Unk") ||
        family.includes("None")
          ? `${fullName} - ${title}`)
          : `${fullName} of House ${family}`;
```

so we don't repeat ourselves